test/lint/check-doc.py

```diff
@@ -26,12 +26,8 @@
 
 
 def main():
-    if sys.version_info >= (3, 6):
-        used = check_output(CMD_GREP_ARGS, shell=True, universal_newlines=True, encoding='utf8')
-        docd = check_output(CMD_GREP_DOCS, shell=True, universal_newlines=True, encoding='utf8')
-    else:
-        used = check_output(CMD_GREP_ARGS, shell=True, universal_newlines=True) # encoding='utf8'
```

Comment:
Let's change only the `< sys.version_info >= (3, 6)` case here, and keep the 3.6+ one for when 3.6 support is the minimum.